documentation/specs/proposed/wasmtask.wit

```diff
@@ -0,0 +1,62 @@
+﻿package msbuildtask:v0
+
+record str-kv-pair {
+    key: string
+    value: string
+}
+
+type str-dictionary = list<str-kv-pair>
+
+record TaskItem {
+    ItemSpec: string
+    Metadata: str-dictionary
+}
+
+type TaskItems = list<TaskItem>
+type Bools = list<bool>
+type Strings = list<string>
+
+interface host-data {
+    GetHostData: func() -> str-dictionary
+}
+
+
+interface host-logger {
```

Comment:
I'd appreciate some summary comments on the interfaces describing their intended use. I understand `host-logger` but not all of the others.